diode/client.go

```diff
@@ -146,6 +174,14 @@ func getAuthRetries(maxAuthRetries int) (int, error) {
 	return maxAuthRetries, nil
 }
 
+// getCertFile returns the cert file path either from provided value or environment variable
+func getCertFile(certFile string) string {
```

Comment:
Perhaps name this function to `getCertPath`?

Reply:
If we rename this, then probably for consistency we should rename `certFile` fields in the structs for consistency?

Reply:
Then for parity we should also rename env variable here and in the already released python SDK, leaning leaving as is for consistency :/